### PR TITLE
OPSEXP-2526: bump alfresco-repository to allow repo hpa and add tests

### PR DESCRIPTION
**Pull request by alxgomz**
Ref: OPSEXP-2526

### CODE REVIEW

File: helm/alfresco-content-services/templates/config-repository.yaml [hunk 1]
```diff
@@ -8,7 +8,6 @@ data:
   alfresco-global.properties: |
     {{- $alfUrl := include "alfresco-common.external.url" . }}
     deployment.method=HELM_CHART
-    alfresco.cluster.enabled={{ gt (index .Values "alfresco-repository" "replicaCount" | int) 1 }}
```
Comment (gionn) [inline helm/alfresco-content-services/templates/config-repository.yaml:11]: where is this gone?

File: test/enterprise-integration-test-values.yaml [hunk 2]
```diff
@@ -150,23 +150,10 @@ alfresco-sync-service:
...
-  aws:
-    accessKeyId: AAAAAAAAAAAAAAAAA
-    secretAccessKey: ZZZZZZZZZZZZZZZZZ
-    s3Bucket: somebucket
-    region: us-east-1
-    comprehendRoleARN: arn:aws:iam::000000000000:user/comprehend
-  resources:
-    requests:
-      cpu: "0.01"
-      memory: "512Mi"
```
Comment (gionn) [inline test/enterprise-integration-test-values.yaml:165]: in this way we are not testing it anymore in acs-deployment

**Review by gionn — CHANGES_REQUESTED** [review]

File: .github/workflows/helm-enterprise.yml [hunk 3]
```diff
@@ -98,49 +99,60 @@ jobs:
...
       - name: Check if cgroup v2 workaround is needed
+        if: startsWith(matrix.values, '7.')
+        env:
+          VALUES_FILE: ${{ matrix.values }}
         id: cgroupv2-workaround-extra-values
         run: |
-          acs_version=$(echo ${{ matrix.values }} | cut -d'.' -f1,2)
-          is_affected=$(echo "$acs_version <= 7.2" | bc -l)
-          if [ "$is_affected" -eq 1 ]; then
+          if [ ${VALUES_FILE:2:1} -le 2 ]; then
```
[the remark this answers is not in the document]
Comment (alxgomz) [reply .github/workflows/helm-enterprise.yml:107]: the goal was mostly to get rid of the warnings for non versioned value files:
https://github.com/Alfresco/acs-deployment/actions/runs/8282284667/job/22662768832#step:9:11

File: test/enterprise-integration-test-values.yaml (hunk 2 above)
Comment (alxgomz) [reply test/enterprise-integration-test-values.yaml:165]: True but this test do not add any value compared to the component chart's tests we run with `ct`

File: .github/workflows/helm-enterprise.yml [hunk 4]
```diff
@@ -98,49 +99,60 @@ jobs:
...
+            --values helm/${{ matrix.name }}/${{ matrix.values }} \
+            --values test/enterprise-integration-test-values.yaml \
+            ${{ steps.cgroupv2-workaround-extra-values.outputs.helm_install_params }} \
+            ${{ steps.configurable-extra-values.outputs.helm_install_params }}
+          kubectl get pods --watch-only &
+          KWPID=$!
+          kubectl wait --timeout=7m --all=true --for=condition=Available deploy && kill $KWPID
+          if [ "${VALUES_FILE:0:2}" != "7." ] ; then echo "Waiting for ESC Reindexing job to complete"
+            kubectl wait --timeout=5m --for=condition=complete job/acs-alfresco-search-enterprise-reindexing
+          fi
```
[the remark this answers is not in the document]
Comment (alxgomz) [reply .github/workflows/helm-enterprise.yml:145]: I can't really do that.
The goal here is to have logs that sequentially showing the pod status changes one after the other. This prove useful when I tried to debug overly long installations. If I restore the previous command the steps will silently install  up to completion and only after wards we spit a status that much harder to understand when it comes to finding which component took more time to spin up.

[the remark this answers is not in the document]
Comment (gionn) [reply .github/workflows/helm-enterprise.yml:145]: I meant reverting just the formatting, I think it's still worth having a step for the helm install (without wait) and another one with the stuff you introduced, along with a step name which describe what we are trying to achieve there

File: test/enterprise-integration-test-values.yaml (hunk 2 above)
Comment (alxgomz) [reply test/enterprise-integration-test-values.yaml:165]: I guess I removed it in an attempt to speed up deployment (before I enabled the sequential logging during installation). If you're really sure we can catch something here I can try to re-enable it (though I'd like to know what we can catch here tbh)

Comment (gionn) [reply test/enterprise-integration-test-values.yaml]: e.g. if we are injecting configmaps in a consistent manner (with the appropriate keys) - which are managed in this chart only

File: .github/workflows/helm-enterprise.yml [hunk 5]
```diff
@@ -98,49 +99,63 @@ jobs:
...
+              --set dtas.additionalArgs[0]='-k not test_renditions'" >> "$GITHUB_OUTPUT"
           fi
           if [ "${{ matrix.values }}" = "7.0.N_values.yaml" ]; then
             echo "Skipping renditions tests on ACS 7.0.N"
             echo "helm_install_params=--set dtas.additionalArgs[0]='-k not test_renditions'" >> "$GITHUB_OUTPUT"
           fi
 
       - name: Helm install
-        run: >-
+        run: |
```
Comment (gionn) [inline .github/workflows/helm-enterprise.yml:128]: revert 🙏🏻

File: helm/alfresco-content-services/Chart.yaml
Comment (gionn) [inline helm/alfresco-content-services/Chart.yaml:1]: should we move also the main chart version to 8.1.0-alpha.0?

File: .github/workflows/helm-enterprise.yml [hunk 6]
```diff
@@ -123,24 +128,35 @@ jobs:
         run: >-
           helm dep up ./helm/alfresco-content-services &&
```
Comment (gionn) [inline .github/workflows/helm-enterprise.yml:129]: you can move this to a dedicated step, but more importantly I see only now that it should be `helm dep build` and not `up` 😱  check also for other usages

Comment (alxgomz) [reply .github/workflows/helm-enterprise.yml]: will be tackled in OPSEXP-2590

**Review by gionn — APPROVED** [review]

**Pull request merged**